### PR TITLE
Blog Post :: Recovering Quorum After Renaming Servers

### PR DESCRIPTION
**Pull request by johnweldon**

### CODE REVIEW

File: content/blog/nats-recover-quorum.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ $ nats server report jetstream
 ╰────────────────────┴────────────────┴─────────┴───────────┴──────────┴───────┴────────┴──────┴─────────┴─────────╯
 
 
-WARNING: No cluster meta leader found. The cluster expects 6 nodes but only 3 responded. JetStream operation require at least 4 up nodes.
+WARNING: No cluster meta leader found. The cluster expects 6 nodes but only 3 responded. JetStream operation requires at least 4 up nodes.
```
Comment (johnweldon) [inline content/blog/nats-recover-quorum.md:126]: ditto

Comment (gcolliso) [reply content/blog/nats-recover-quorum.md:126]: Well, this is still grammatically incorrect. That warning should be updated 😉

Comment (johnweldon) [reply content/blog/nats-recover-quorum.md:126]: https://github.com/nats-io/natscli/pull/974

**Comment by gcolliso** [comment]
lgtm - thanks @johnweldon

**Pull request merged**